### ISSUE TITLE
Override ENV TransformFunctions 

### ISSUE THREAD

**Issue by jeff-bradley** [CrunchyData/pg_featureserv#172]
Is it possible to override the TransformFunctions parameter with PGFS_SERVER_TRANSFORMFUNCTION?

I have tried multiple ways of setting the variable but nothing seems to stick, the only way I can set them is to uncomment it in the config.yaml file.

I have tried locally using windows and also in GCP on Cloud Run setting environment variables. Other parameters like limitmax all see to work.

**Comment by dr-jts** [comment]
The environment variable name should be `PGFS_SERVER_TRANSFORMFUNCTIONS`.

Using the following syntax is working for me:
```
export PGFS_SERVER_TRANSFORMFUNCTIONS="ST_Boundary,ST_Centroid,ST_Extent"
```

**Comment by jeff-bradley** [comment]
weird, I still cant get the override to work. I tried locally on windows and wsl, I also tried on GCP Cloud Run all using the variable PGFS_SERVER_TRANSFORMFUNCTIONS. The transform function works just fine if I manually alter the config yaml, so could just be something with the operating systems, I have tried not setting this variable correctly. I am ultimately able to solve my goal by enabling the transform functions in cloud run following this https://github.com/CrunchyData/pg_featureserv/issues/151#issuecomment-1821592215
thanks!
